vizro-core/docs/pages/user-guides/install.md

```diff
@@ -1,61 +1,146 @@
+**Python**: Vizro supports macOS, Linux, and Windows. It works with Python 3.8 and later. You can specify the version of Python to use with Vizro when you set up a virtual environment.
 
 
-??? tip "Beginners/Code novices"
-    If you consider yourself a beginner or coding novice, you can follow the steps in these boxes to completely avoid the terminal usage
+**Virtual environment**: You should create a virtual environment for each Vizro project you work on to
+  isolate its Python dependencies from those of other projects. See the following references to learn more about [Python virtual environments](https://realpython.com/python-virtual-environments-a-primer/), [Conda virtual environments](https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html#starting-conda) or [watch an explainer video about them](https://youtu.be/YKfAwIItO7M).
 
-    - install the [Anaconda Navigator](https://www.anaconda.com/download)
-    - create a new environment as [outlined here](https://docs.anaconda.com/free/navigator/tutorials/manage-environments/), choose a Python version `>=3.8`
+!!! tip "Prefer to use Vizro without opening a terminal window?"
 
-## Install
+    If you are a beginner or coding novice, you may prefer to avoid using a terminal to work with Vizro. Skip to the ["Use Vizro inside Anaconda" section](#use-vizro-inside-anaconda-navigator) below.
 
-To install Vizro from the Python Package Index (PyPI), use [`pip`](https://pip.pypa.io/en/stable/) in your terminal window:
+
+
+??? information "How to create a virtual environment for your Vizro project"
+
+    ### Using `conda`
+    We strongly recommend [installing `conda` as your virtual environment manager](https://docs.conda.io/projects/conda/en/latest/user-guide/install/) if you don't already use it. Once installed, create a virtual environment from the terminal as follows:
+
+    ```bash
+    conda create --name vizro-environment python=3.10 -y
+    ```
+
+    (This example uses Python 3.10, and creates a virtual environment called `vizro-environment`. You can opt for a different version of Python (any version >= 3.8 and <3.12), and name it anything you choose).
+
+    Activate your virtual environment from any directory:
+
+    ```bash
+    conda activate kedro-environment
+    ```
+
+    To confirm that a valid version of Python is installed in your virtual environment, type the following in your terminal (macOS and Linux):
+
+    ```bash
+    python3 --version
+    ```
+
+    On Windows:
+
+    ```bash
+    python --version
+    ```
+
+    ??? information "How to create a virtual environment without using conda"
+
+        ### How to create a new virtual environment without using `conda`
+
+        You can alternatively create virtual environments using `venv` or `pipenv` instead of `conda`.
+
```

Comment:
Are these double collapsible panels intended? It kind of looks a bit funny. Better to add as a collapsible after?

<img width="777" alt="Screenshot 2024-02-27 at 14 14 42" src="https://github.com/mckinsey/vizro/assets/90609403/5fd48bbd-0566-4485-b9f9-7d478f3929cf">

Reply:
They are intentional. I initially thought they worked quite well but maybe they are somewhat gnarly. Let's see what general consensus is!

Reply:
I would have initially said let's avoid it, but having tested it live, I am actually fine with them. In general I like when information that might not be needed is hidden away.

So happy to keep bit, but no strong opinion

Reply:
Hmmmm personally I am slightly against it but not a strong opinion, so happy to go with whatever the majority is here.

Reply:
I have a stronger preference against it :P I think from a design perspective, they look a bit funky. When it comes to admonitions I am also following this editorial guide: https://ocular-d.github.io/styleguide-editorial/admonitions.html#tip

And one of their points is: **Don't nest admonitions.** 😄 

Regardless of the editorial style / design - I also think that in this case it's not really suitable in terms of content? Using env or pipenv is for me an equal alternative to conda. So I don't see a reason why that alternative should be hidden in another collapsible. There is no hierarchy for me between these two approaches. 

However, if you all have a strong preference for it, I would adapt 😄

Reply:
Let's not make you adapt! I am still on the fence too -- I like it but I'm also unsure it's best practice. Let me back it out and use horizontal spacers to break up the text.